src/actions/dot/migrate/index.ts

```diff
@@ -129,13 +142,13 @@ const migrate = async (opts: Options) => {
   let wait = false;
   for (const block of blocks) {
     const { extrinsics, number } = block;
-    console.log(`Checking block ${number}`);
+    // console.log(`Checking block ${number}`);
```

Comment:
I like this logging because it's informational when running the script. Can we keep this as logger.log("info", msg) ?